### ISSUE TITLE
Epic: Slack Notification / Bot Service

### ISSUE THREAD

**Issue by slu-it** [nt-ca-aqe/library-app#9]

**Comment by antoniq** [comment]
Small update regarding Spring Cloud Contract with PACT - we should follow this issue for Kotlin support, until it is not released we cannot use Spring Cloud Contract: https://github.com/spring-cloud/spring-cloud-contract/issues/434

**Comment by marcingrzejszczak** [comment]
Why is Kotlin a prerequisite? You can use YAML or Groovy

**Comment by antoniq** [comment]
Because the Provider is written in Kotlin and as far as I know, the integration with Spring Cloud Contract (for example for autogenaring tests) is still work in progress.

**Comment by marcingrzejszczak** [comment]
Yes, it is in progress, but you can generate Java tests with a YAML contract. Nothing stops you from using contract tests defined like this.
